apps/web/modules/survey/components/question-form-input/index.test.tsx

```
@@ -101,6 +101,23 @@ vi.mock("@/lib/hashString", () => ({
 
+// Mock utils from question-form-input
+vi.mock("./utils", () => ({
+  determineImageUploaderVisibility: () => false,
+  getChoiceLabel: () => ({ default: "Choice Label", fr: "Choice Label FR" }),
+  getEndingCardText: () => ({ default: "Ending Text", fr: "Ending Text FR" }),
+  getWelcomeCardText: () => ({ default: "Welcome Text", fr: "Welcome Text FR" }),
+  getMatrixLabel: () => ({ default: "Matrix Label", fr: "Matrix Label FR" }),
+  getPlaceHolderById: () => "Placeholder Text",
+  getIndex: (id: string) => {
+    if (id.includes(".")) {
+      return parseInt(id.split(".")[1]);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)